go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -882,6 +892,51 @@ func (c *TabletConfig) Verify() error {
 	return nil
 }
 
+// verifyUnmanagedTabletConfig checks unmanaged tablet related config for sanity
+func (c *TabletConfig) verifyUnmanagedTabletConfig() error {
+	// Skip checks if tablet is not unmanaged
+	if !c.UnmanagedTablet {
+		return nil
+	}
+
+	// Throw error if both host and socket are null
+	if !c.DB.HasGlobalSettings() {
+		return errors.New("no connection parameters specified but unmanaged mode specified")
+	}
+	if c.DB.App.User == "" {
+		return errors.New("database app user not specified")
+	}
+	if c.DB.App.Password == "" {
+		return errors.New("database app user password not specified")
+	}
```

Comment:
These two are not sufficient. All the users and passwords should be specified. Though it is possible that if you aren't planning to import data aka MoveTables, you don't need the other users.

Reply:
So, if it is possible for the remaining to be empty, we don't need any additional validations?

Reply:
I don't know. Let's start with this, and we can add more later.